notes/templates/core/edit_note.html

```diff
@@ -0,0 +1,13 @@
+<h1>Edit Note</h1>
```

Comment:
You want all your templates other than `base.html` to extend that template

`{% extends 'base.html' %}`

Reply:
And then wrap the content in `{% block content %}`...your html...`{% endblock %}`